libs/application/template-api-modules/src/lib/modules/templates/new-primary-school/new-primary-school.utils.ts

```diff
@@ -289,7 +294,10 @@ export const transformApplicationToNewPrimarySchoolDTO = (
         alternativeSpecialEducationDepartmentIds.length > 0 && {
           alternativeOrganizationIds: alternativeSpecialEducationDepartmentIds,
         }),
-      requestingMeeting: requestingMeeting === YES,
+      ...(!isSpecialEducation &&
+        applicationType !== ApplicationType.CONTINUING_ENROLLMENT && {
+          requestingMeeting: requestingMeeting === YES,
+        }),
```

Comment:
Add `shouldShowPage()` check so it's the same condition used in supportSubSection?

```suggestion
      ...(shouldShowPage(...) && !isSpecialEducation &&
        applicationType !== ApplicationType.CONTINUING_ENROLLMENT && {
          requestingMeeting: requestingMeeting === YES,
        }),
```